bin/functions/build.sh

```diff
@@ -51,16 +51,18 @@ EOH
 
 function build_uberjar_proj {
     PROJ=$1
-    echo "Building '$PROJ' uberjar ..."
+    cprintln $RED2$GREEN$UNDERLINE "Building '$PROJ' uberjar ..."
+    
     if [ "$CMR_INTERNAL_NEXUS_REPO" = "true" ]; then
+        cprintln $RED$BLINK "with nexus"
         (cd $CMR_DIR/$PROJ && \
             lein with-profile +uberjar,+internal-repos do clean, uberjar)
     else
-        (cd $CMR_DIR/$PROJ && \
+        (cd $CMR_DIR/$PROJ && pwd && \
```

Comment:
is pwd debug here?

Reply:
that's a you question tbh, I didn't add that in

Reply:
Pulled it out, did indeed seem to be debug. Wasn't adding much of use